src/ras2fim.py

```diff
@@ -552,8 +562,8 @@ def create_input_args_log(**kwargs):
     #            -n E:\X-NWS\X-National_Datasets -r "C:\Program Files (x86)\HEC\HEC-RAS\6.3"
```

Comment:
I believe this line needs to be tweaked slightly to include the new required inputs